pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -373,6 +367,14 @@ func (t *clusterCache) Key() string {
 		sort.Strings(nv)
 		params = append(params, nv...)
 	}
+	if t.proxyMetadata != nil {
+		md := make([]string, 0, len(t.proxyMetadata))
+		for mk, mv := range t.proxyMetadata {
+			md = append(md, mk+"~"+mv)
+		}
```

Comment:
It can be a huge map, can you add the one that really matters

Reply:
currently certificates matter. But not sure what else matters - I can trim down to certs but do we know/sure we do not care about any thing else?